third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -583,6 +583,30 @@ settings.backup_configuration.binary_log_enabled are both set to true.`,
 				Computed:    true,
 				Description: `The URI of the created resource.`,
 			},
+			"restore_backup_context": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"backup_run_id": {
+							Type:        schema.TypeInt,
+							Required:    true,
+							Description: `The ID of the backup run to restore from.`,
+						},
+						"instance_id": {
```

Comment:
Just curious: what does specifying this field (or not specifying this field) change?

Reply:
You can optionally restore to the backup of another instance (backup ID's are unique per instance). If left blank, the API just assumes the same instance. I can add a line in the documentation to make this more clear.